.github/ISSUE_TEMPLATE/release-it-tests.md

```diff
@@ -0,0 +1,41 @@
+|||
+| :--                                   | :--                                                       |
+| **Main release issue**                | RELEASE_ISSUE                                             |
+| **Main release candidate issue**      | RELEASE_CANDIDATE_ISSUE_URL                               |
+| **Version**                           | WAZUH_VERSION                                             |
+| **Release candidate**                 | RC_VERSION                                                |
+| **wazuh/wazuh tag**                   | https://github.com/wazuh/wazuh/tree/vWAZUH_VERSION-TAG    |
+| **wazuh/wazuh-qa tag**                | https://github.com/wazuh/wazuh/tree/vWAZUH_VERSION-TAG    |
```

Comment:
Wrong url for wazuh-qa

Reply:
Does it make sense? we are using always paired branch naming

Reply:
Yes, because the tag is used to launch the integration tests during releases (the main purpose of this template)